src/Phinx/Migration/Manager.php

```diff
@@ -368,61 +391,89 @@ public function executeSeed($name, SeedInterface $seed)
      * Rollback an environment to the specified version.
      *
      * @param string $environment Environment
-     * @param int $version
+     * @param int $target
      * @param bool $force
+     * @param bool $targetMustMatchVersionName
      * @return void
      */
-    public function rollback($environment, $version = null, $force = false)
+    public function rollback($environment, $target = null, $force = false, $targetMustMatchVersionName = true)
     {
-        $migrations = $this->getMigrations();
-        $versionLog = $this->getEnvironment($environment)->getVersionLog();
-        $versions = array_keys($versionLog);
+        // note that the migrations are indexed by name (aka creation time) in ascending order
+        $projectMigrations = $this->getMigrations();
```

Comment:
Don't rename existing variables unless the intent has changed.

`$migrations = $this->getMigrations();` is fine.